### ISSUE TITLE
Unexpected error while importing SFFCorrector from lightkurve (via eleanor) after latest update

### ISSUE THREAD

**Issue by rachelbf** [lightkurve/lightkurve#975]
#### Problem description

I was initially having an issue while using lightkurve's bin() via another package called triceratops (see below) and was encouraged to update to the latest lightkurve version to fix that. 
![Screen Shot 2021-02-22 at 16 54 57](https://user-images.githubusercontent.com/48811452/108902725-b631c480-75d9-11eb-831a-a01269e901db.png)

After updating lightkurve, I got the error in the example below. This error didn't exist before updating.

#### Example

![Screen Shot 2021-02-23 at 13 12 51](https://user-images.githubusercontent.com/48811452/108902309-2e4bba80-75d9-11eb-937e-20647dc9613a.png)

#### Environment

-  platform (e.g. Linux, OSX, Windows): Mac OS Catalina
-  lightkurve version (e.g. 1.0b6): 2.0.2
-  installation method (e.g. pip, conda, source): pip
-  python version: 3.7
-  eleanor version: 2.0.2

**Comment by barentsen** [comment]
@rachelbf Thank you for opening an issue!  We released Lightkurve v2 last week. Among several other changes, we reduced the amount of code that is executed on the initial import of Lightkurve.

Unfortunately, it means that this line:
```python
from lightkurve import SFFCorrector
```
needs to be replaced with 
```python
from lightkurve.correctors import SFFCorrector
```

I'm happy to consider undo-ing this change if it breaks a significant amount of people's code.

I hope this helps?  Please let me know if you encounter additional errors.

**Comment by rachelbf** [comment]
@barentsen Thank you for the prompt reply. So far, eleanor is the only code that I use that uses lightkurve and broke, but I can easily bring this to their attention.
Thanks once again!

**Comment by barentsen** [comment]
@rachelbf We just released Lightkurve v2.0.3 which re-enables imports such as `from lightkurve import SFFCorrector`.

This means `eleanor` should work again after upgrading Lightkurve (e.g. `python -m pip install lightkurve --upgrade`).

Thanks again! :+1:
